### PR TITLE
fix pg-cfpages-basic wrangler dep

### PR DESCRIPTION
**Pull request by Jolg42**
See [internal Slack](https://prisma-company.slack.com/archives/CRGDU19K6/p1706796179696709)

### CODE REVIEW

File: driver-adapters-wasm/pg-cfpages-basic/run.sh [hunk 1]
```diff
@@ -7,6 +7,6 @@ pnpm install
 pnpm prisma generate
 
 pnpm wrangler pages functions build --node-compat --outdir build/
-pnpm wrangler publish build/index.js --name pg-cfpages-basic public
+pnpm wrangler publish build/index.js --name pg-cfpages-basic
```
Comment (millsp) [inline driver-adapters-wasm/pg-cfpages-basic/run.sh]: Ideally, we would publish it as a page as well?

Comment (Jolg42) [reply driver-adapters-wasm/pg-cfpages-basic/run.sh]: Yeah, this is not working

Comment (Jolg42) [reply driver-adapters-wasm/pg-cfpages-basic/run.sh]: I'll leave this aside for now

File: driver-adapters-wasm/pg-cfpages-basic/run.sh [hunk 2]
```diff
@@ -6,5 +6,7 @@ pnpm install
 
 pnpm prisma generate
 
-pnpm wrangler pages deploy . --project-name pg-cfpages-basic --node-compat | tee deployment-logs.txt
-sleep 15
\ No newline at end of file
+pnpm wrangler pages functions build --node-compat --outdir build/
+pnpm wrangler publish build/index.js --name pg-cfpages-basic
```
Comment (janpio) [inline driver-adapters-wasm/pg-cfpages-basic/run.sh]:
```suggestion
pnpm wrangler publish build/index.js --name pg-cfpages-basic --compatibility-date 2024-02-01
```

**Pull request closed without merging**